randovania/games/planets_zebeth/exporter/patch_data_factory.py

```diff
@@ -1,12 +1,136 @@
+            res_lock = pickup.original_pickup.resource_lock
+            text_index = (
+                1
+                if (
+                    res_lock is not None
+                    and "Locked" in res_lock.temporary_item.long_name
+                    and len(pickup.collection_text) > 1
+                )
+                else 0
+            )
+
+            pickup_map_dict[object_id] = {
+                "model": pickup.model.name,
+                "type": pickup.original_pickup.name if not pickup.other_player else "Nothing",
+                "quantity": quantity,
+                "text": {
+                    "header": item_info[pickup.name]["text_header"]
+                    if not self.players_config.is_multiworld
+                    else pickup.name,
+                    "description": pickup.collection_text[text_index],
+                },
+            }
+
+        return pickup_map_dict
+
+    def _create_starting_items_dict(self) -> dict:
+        starting_resources = self.patches.starting_resources()
+        return {resource.long_name: quantity for resource, quantity in starting_resources.as_resource_gain()}
+
+    def _create_starting_memo(self) -> str:
+        starting_memo = None
+        extra_starting = item_names.additional_starting_equipment(self.configuration, self.game, self.patches)
+        if extra_starting:
+            starting_memo = ", ".join(extra_starting)
+        return starting_memo
+
+    def _create_starting_location(self) -> dict:
+        return {
+            "x": self.game.region_list.node_by_identifier(self.patches.starting_location).extra["global_x"],
+            "y": self.game.region_list.node_by_identifier(self.patches.starting_location).extra["global_y"],
+        }
+
+    def _create_hash_dict(self) -> dict:
+        return {
+            "word_hash": self.description.shareable_word_hash,
+            "hash": self.description.shareable_hash,
+            "session_uuid": str(self.players_config.get_own_uuid()),
+        }
+
+    def _create_game_config_dict(self):
+        return {
+            "starting_room": self._create_starting_location(),
+            "seed_identifier": self._create_hash_dict(),
+            "starting_items": self._create_starting_items_dict(),
+            "starting_memo": self._create_starting_memo(),
+        }
+
+    def _create_cosmetics(self) -> dict:
+        c = self.cosmetic_patches
+        return {
+            "show_unexplored_map": c.show_unexplored_map,
+            "room_names_on_hud": c.show_room_names.value,
+        }
+
+    def _get_item_data(self):
+        item_data = json_lib.read_path(
+            RandovaniaGame.METROID_PLANETS_ZEBETH.data_path.joinpath("pickup_database", "item_data.json")
+        )
+
+        item_data["Missiles"] = item_data["Missile Tank"]
+        item_data["EnergyTransferModule"] = item_data["Nothing"]
+        return item_data
+
+    def _create_object_datas(self):
+        pass
```

Comment:
what is this for? It's not referenced anywhere, I cant see it being a parent function in PDF.

Reply:
No idea and removed it since it doesn't seem to be used